packages/@headlessui-react/src/components/combobox/combobox.tsx

```diff
@@ -1663,6 +1663,14 @@ function OptionsFn<TTag extends ElementType = typeof DEFAULT_OPTIONS_TAG>(
     actions.setActivationTrigger(ActivationTrigger.Pointer)
   })
 
+  // When the user scrolls **using the native scrollbar**, the only event fired is
```

Comment:
The bug appears to only occur on desktop, so we're ignoring scroll reactions on mobile. An alternative approach could be to maintain state around ﻿`mousedown`, ﻿`mouseup`, and ﻿`onScroll` events, but this method allows us to avoid adding any additional ﻿`useState` functions.